enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,389 @@
+and direct-attached storage. The internal networks are fenced but offer connectivity options to IBM Cloud infrastructure
+or on-premises environments. Both public and private network is supported for instances (VMs). For public networks, only
+limited ports are opened for inbound access and this is not configurable.
+
+Basic Walkthrough:
+1. The user creates a paid IBM Cloud Account (Power VS - or any infrastructure service, is not available for free accounts).
+2. The user creates a Power VS service in a specific region.
+3. The user creates a [DNS domain[](#dns) in CIS.
+4. The installer creates a private network in the Power VS service via terraform.
+5. The installer [uploads/creates<sup>*</sup>](#coreos-ova) the required boot image via terraform.
+6. The installer creates VMs according to specified [configurable](#install-config) values, which are configured via
+terraform, bootstrapping, or in-cluster mechanisms such as MAPI:
+- \# of CPU / Cores
+- Shared vs. Dedicated CPUs
+- [Machine type (s922 vs. e980)][power-cloud-instances]
+- \# of Memory
+- Boot Image [set in installer binary]
+- Network (public, private) [optional]
+- Additional disk volume(s)
+- Storage Type (Tier 1 or 3)
+
+<sup>*</sup>The `Disk capacity` property of the image as defined in the OVA is the size of the final boot image once
+imported. The size and capacity can be different, and we're planning to keep the size as 16G with a default capacity of
+120G. Just a note, the 16G refers to the raw disk file within the OVA, not the boot image.
+
+[Power VS Reference Doc][power-vs-reference-doc]
+
+### Installation Configuration
+Power VS doesn't use cloud instance "flavors" with pre-defined cpus, memory, and storage.
+Instead, we will have some defaults set in the installer. Storage will default to 120Gb to match the OVA disk image.
+Both CPU and memory also have defaults, but can also be overridden in the install config. There are also options
+for processor type and system type.
+
+You can override these like so:
+```yaml
+compute:
+- architecture: ppc64le
+  hyperthreading: Enabled
+  name: worker
+  platform:
+  - processors: 0.25
+  - memory: 8
+  - procType: shared
+  - sysType: s922
+  replicas: 3
+controlPlane:
+  architecture: ppc64le
+  hyperthreading: Enabled
+  name: master
+  platform:
+  - processors: 0.25
+  - memory: 8
+  - procType: shared
+  - sysType: s922
+  replicas: 3
+```
+
+When you're installing, you'll also need to specify an API key to authenticate. This is done by specifying the IBM_APIKEY
+environment variable. This is then written to disk, and used again later. For a cluster destroy, this key should be set
+locally using that environment variable as well.
+
+Finally, here is a full sample of an **install-config.yaml** for a Power VS installation.
+```yaml
+baseDomain: powervs-ibm.com
+compute:
+- architecture: ppc64le
+hyperthreading: Enabled
+name: worker
+platform: {}
```

Comment:
This seems redundant with lines 201 and 215. Is that a copy-and-paste error in duplicating the section? Or an indentation issue with the PR (or the way it's rendered in the diff view)?

Reply:
Now with 100% (probably) better indentation :wink: